### ISSUE TITLE
Missing ANDROID_HOME environment variable

### ISSUE THREAD

**Issue by chovanjo** [beevelop/docker-cordova#68]
Hi Beevelop,

There is missing the ANDROID_HOME environment variable.

```
$ npx cordova build
Checking Java JDK and Android SDK versions
ANDROID_HOME=undefined (recommended setting)
ANDROID_SDK_ROOT=/opt/android (DEPRECATED)
Failed to find 'ANDROID_HOME' environment variable. Try setting it manually.
Detected 'avdmanager' command at /opt/android/cmdline-tools/bin but no 'tools/bin' directory found near.
Try reinstall Android SDK or update your PATH to include valid path to SDK/tools/bin directory.
```

**Comment by patrickkurth** [comment]
Got the same issue. Did you find a workaround? 

I tried using the underlying images [docker-android-nodejs](https://github.com/beevelop/docker-android-nodejs) and [docker-android](https://github.com/beevelop/docker-android) with the same result. So I think the problem comes from the docker-android image, maybe due to the deprecation of "ANDROID_SDK_ROOT". I will investigate this a bit further.

**Comment by beevelop** [comment]
`ANDROID_HOME` should be fixed in the base images now. There was a remaining issue with the build-tools being to new for Cordova. I hope the issue is resolved from `v2023.04.1` onwards. Feedback is highly appreciated as I am not using this image personally anymore. 🙏

**Comment by chovanjo** [comment]
Hi,

I have fixed my issues like this 

```
FROM beevelop/cordova:latest

ENV ANDROID_HOME=/opt/android
RUN while true; do echo 'y'; sleep 2; done | sdkmanager "build-tools;32.0.0"
```
